.github/ISSUE_TEMPLATE/Epic.md

```diff
@@ -0,0 +1,42 @@
+---
+name: Epic template
+about: Template for creating epics
+title: "[EPIC]"
+labels: Epic
+assignees: ''
+
+---
+
+<!--
+An Epic should define a higher-level package of work or project iteration that, once finished, makes sense to merge in core.
+-->
+## Description ##
+<!-- The what -->
+
+## Rationale ##
+<!-- The why -->
+
+## Stakeholders ##
+<!-- The who. This could include developers, designers, accessibility folks, testers, devrel, PMs... -->
+
+## Dependencies & Other Context ##
+<!--
+Link here any relevant Overview issues or posts that provide context
+-->
+
+## Acceptance Criteria & Required Scope ##
```

Comment:
I'd just make this "Tasks" perhaps. Reduces the complexity a bit, and makes it a bit easier to understand what you should add.

Reply:
What would be an easier way to say `Acceptance criteria`? `Required tasks`? What I want to emphasize is that any task listed in this section is absolutely necessary: if a single one is missing, the feature is not ready, whereas if all of them are finished, it means it's ready for inclusion in core.